AppleStore/AppleStore/MyViewController.swift

```diff
@@ -9,9 +9,233 @@ import UIKit
 
 class MyViewController: UIViewController {
```

Comment:
```suggestion
final class MyViewController: UIViewController {
```
в 1001 раз не поставил final - 10 присевов